### ISSUE TITLE
socket connected, but

### ISSUE THREAD

**Issue by Somnius** [dante4rt/getgrass-bot#41]
as all other bots, this one too connects using the script, no proxies needed, direct and doesn't apear connected on the dashboard.

all bots are missing the bcrypt methods to apear as token making machines.. so why bother, really, to sell proxies?

**Comment by lakompee** [comment]
yeah me too, bot connected and running, but no connection on the dashboard
